composer.json

```diff
@@ -30,22 +30,22 @@
     "require": {
         "php": "^7.2",
 
-        "typo3/cms-backend": "^10.4.1 || ~11.0.0",
-        "typo3/cms-core": "^10.4.1 || ~11.0.0",
-        "typo3/cms-extbase": "^10.4.1 || ~11.0.0",
-        "typo3/cms-extensionmanager": "^10.4.1 || ~11.0.0",
-        "typo3/cms-fluid": "^10.4.1 || ~11.0.0",
-        "typo3/cms-frontend": "^10.4.1 || ~11.0.0",
-        "typo3/cms-install": "^10.4.1 || ~11.0.0",
+        "typo3/cms-backend": "^10.4.1 || ^11.0.0 ||  ^11.1.0",
```

Comment:
please provide `>=11.0.0 <11.2` as dependency constraint

Reply:
ext_emconf.php and extension composer.json needs to be adapted as well to match these changes here